tests/default/flow_framework/workflow.yaml

```
@@ -49,11 +119,79 @@ chapters:
         name: test_update_work_flow
     response:
       status: 404
+  - synopsis: Get workflow steps.
+    path: /_plugins/_flow_framework/workflow/_steps
+    method: GET
+    response:
+      status: 200
+  - synopsis: Get workflow steps with bad request.
+    path: /_plugins/_flow_framework/workflow/_steps
+    method: GET
+    parameters:
+      workflow_step: wrong_query
+    response:
+      status: 400
+  - synopsis: Get workflow status.
+    path: /_plugins/_flow_framework/workflow/{workflow_id}/_status
+    method: GET
+    parameters:
+      workflow_id: ${create_flow_framework.test_workflow_id}
+    response:
+      status: 200
+  - synopsis: Get workflow status with wrong workflow ID.
+    path: /_plugins/_flow_framework/workflow/{workflow_id}/_status
+    method: GET
+    parameters:
+      workflow_id: test
```

```suggestion
      workflow_id: invalid
```

Just a nit to be more explicit.